eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1,137 +1,237 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <FindBugsFilter>
-      <!-- These existing KeyVault Attribute APIs return their super class rather than the actual type. -->
```

Comment:
Our .editorconfig has a rule that .xml files should use an indentation of 2 spaces. What editor are you using? IntelliJ should automatically pick up these settings.
https://github.com/Azure/azure-sdk-for-java/blob/master/.editorconfig#L15

Reply:
I am using Intellij. I will edit it and change to 2 space indentation.